frontend/packages/kubevirt-plugin/integration-tests/tests/mocks/mocks.ts

```diff
@@ -63,7 +63,7 @@ deepFreeze(multusNAD);
 
 export const dataVolumeManifest = ({ name, namespace, sourceURL, accessMode, volumeMode }) => {
   return {
-    apiVersion: 'cdi.kubevirt.io/v1alpha1',
+    apiVersion: 'cdi.kubevirt.io/v1beta1',
```

Comment:
I have already done this here for all places (also tests) and we decide to wait for 4.7. https://github.com/openshift/console/pull/6739

Maybe it would be better to merge it atomically? I know there are some bugs because of that atm.

Reply:
If we wait for 4.7, then the 4.6 and 4.6.z tests cannot get PASS. My opinion is let's fixing tests firstly here.

Reply:
ok, so please replace all the occurences in the tests

Reply:
done